<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>811</width>
      <height>750</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>590</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="AI_Turn_Label">
       <property name="text">
        <string>AI Simulate Games</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Simulate_Slide_Bar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="Simulate_Text_Display"/>
     </item>
     <item>
      <widget class="QPushButton" name="Simulation_Start_Button">
       <property name="text">
        <string>Start Simulation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="Player1_Text_Display"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Player1_Train_Soilder_Button">
         <property name="text">
          <string>Train Soldier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Player1_Quantity_Indicator_Label">
         <property name="text">
          <string>Quantity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Player1_Soilder_Persentage_Bar">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>200</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTextBrowser" name="Player2_Text_Display"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Player2_Train_Soilder_Button">
         <property name="text">
          <string>Train Soldier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Player2_Quantity_Indicator_Label">
         <property name="text">
          <string>Quantity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Player2_Soilder_Persentage_Bar">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="Multiplayer_Mode_Button">
       <property name="text">
        <string>Multiplayer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Singleplayer_Mode_Button">
       <property name="text">
        <string>Singleplayer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Reset_Button">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>470</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Start_Button">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>420</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="Turn_Indicator_Label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Turn:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
